build/bundle.js

```diff
@@ -144,7 +144,9 @@ var FCC_Global =
 	// the !- prefixes are for process arguments respective of plugins
 	// Example: https://stackoverflow.com/a/42440360/3530394
 	// style-loader injects css loaded by css-loader through this import statement.
-	var assert = exports.assert = _chai2.default.assert; /* global project_name, projectName */
+	var assert = exports.assert = _chai2.default.assert;
+	// Webpack is configured to load those files with the .html extension as Strings
+	/* global project_name, projectName */
 
```

Comment:
something about the webpack build process that re-arranges the order of comments here, but the order doesn't matter here in bundle.js.